### PR TITLE
Get out

### PR DESCRIPTION
**Pull request by jun-k**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1597)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: japanese/chasitsu.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 な風景を描いた水墨画と、一輪の水仙を入れた花器が飾られている。
 
 [取り敢えずビールを頼む？](beer/beer.md)
+[庭に出る](garden/garden.md)
```
Comment (casey-collab) [inline japanese/chasitsu.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :heavy_check_mark:

**Pull request merged**